.github/workflows/cairo-update-check.yml

```diff
@@ -0,0 +1,49 @@
+name: Cairo Update Check
+
+on:
+  workflow_dispatch:
+    inputs:
+      slack:
+        description: "Send Slack notification on failure"
+        type: boolean
+        default: true
+  schedule:
+    - cron: '0 0 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: software-mansion/setup-scarb@v1
+        with:
+          scarb-version: nightly
```

Comment:
Are we sure scarb will release nightly on daily basis? Why is Scarb even here?

Reply:
discussed offline